docs/topics/forms/media.txt

```diff
@@ -271,29 +303,47 @@ Or if :mod:`~django.contrib.staticfiles` is configured using the
     <script src="https://static.example.com/animations.27e20196a850.js"></script>
     <script src="https://othersite.com/actions.js"></script>
 
+.. _form-media-asset-objects:
+
 Paths as objects
 ----------------
 
-Asset paths may also be given as hashable objects implementing an
-``__html__()`` method. The ``__html__()`` method is typically added using the
-:func:`~django.utils.html.html_safe` decorator. The object is responsible for
-outputting the complete HTML ``<script>`` or ``<link>`` tag content:
+Assets may also be object-based, using :class:`.CSS` and :class:`.JS`.
+Furthermore, these allow you to pass custom HTML attributes::
 
-.. code-block:: pycon
+    class Media:
+        css = {
+            "all": [CSS("pretty.css", media="all", crossorigin="anonymous")],
+        }
```

Comment:
Based off the note where ```The ``media`` attribute on the :class:`.CSS` object overrides the dictionary key.```
I think we're missing a test when media is not provided and when media is different to the dictionary key

Reply:
Excellent catch. The node was actually wrong. I added a test and updated the note to reflect the current behavior.